.idea/$PRODUCT_WORKSPACE_FILE$

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think you added this file by accident. Same for the rest of the `.idea` files.